task/sql-tasks.js

```diff
@@ -248,7 +392,20 @@ async function task_1_18(db) {
  *
  */
 async function task_1_19(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT * FROM 
+            (SELECT 
+                Orders.CustomerID as "CustomerID",
+                Customers.CompanyName as "CompanyName",
+                SUM(UnitPrice*Quantity) as "TotalOrdersAmount, $"
+            FROM OrderDetails
+            INNER JOIN Orders ON OrderDetails.OrderID = Orders.OrderID 
+            INNER JOIN Customers ON Orders.CustomerID = Customers.CustomerID
+            GROUP BY Orders.CustomerID) t 
+        WHERE t.\`TotalOrdersAmount, $\` > 10000
+        ORDER BY t.\`TotalOrdersAmount, $\` DESC,t.\`CustomerID\`

```

Comment:
The query has 2 levels which means for MySQL, as a rule, that WHERE filter will be applied to temporary table with whole records in RAM. Which can be really slow. Please try to make the query in one level SELECT